WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/BackupDownloadStep.kt

```diff
@@ -0,0 +1,31 @@
+package org.wordpress.android.ui.jetpack.backup
+
+import org.wordpress.android.util.wizard.WizardStep
+import javax.inject.Inject
+import javax.inject.Singleton
+
+enum class BackupDownloadStep : WizardStep {
+    DETAILS, PROGRESS, COMPLETE;
+
+    companion object {
+        fun fromString(input: String): BackupDownloadStep {
```

Comment:
Minor (🔍 ): Consider simplifying this function to something like: `fun fromString(input: String) = when (input) { ... }`

Reply:
in 6448ff8